### PR TITLE
Stabilize MSC2659 support

### PR DESCRIPTION
**Pull request by tulir**
https://github.com/matrix-org/matrix-spec-proposals/pull/2659 will hopefully not take too long to get the last tick and go through FCP

### CODE REVIEW

File: changelog.d/15528.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Stabilized support for [MSC2659](https://github.com/matrix-org/matrix-spec-proposals/pull/2659): application service ping endpoint. Contributed by Tulir @ Beeper.
```
Comment (clokep) [inline changelog.d/15528.feature:1]: I think we usually use present tense?

```suggestion
Stabilize support for [MSC2659](https://github.com/matrix-org/matrix-spec-proposals/pull/2659): application service ping endpoint. Contributed by Tulir @ Beeper.
```

File: synapse/appservice/api.py [hunk 2]
```diff
@@ -326,7 +326,7 @@ async def ping(self, service: "ApplicationService", txn_id: Optional[str]) -> No
         assert service.hs_token is not None
 
         await self.post_json_get_json(
-            uri=f"{service.url}{APP_SERVICE_UNSTABLE_PREFIX}/fi.mau.msc2659/ping",
+            uri=f"{service.url}{APP_SERVICE_PREFIX}/ping",
```
Comment (clokep) [inline synapse/appservice/api.py:329]: I'm assuming we don't really care about supporting backwards compatibility at all?

**Review by clokep — APPROVED** [review]

File: synapse/appservice/api.py (hunk 2 above)
Comment (tulir) [reply synapse/appservice/api.py:329]: Yeah, the unstable client endpoint and /versions flag were dropped too, so no need for unstable ping to AS

Comment (clokep) [reply synapse/appservice/api.py:329]: Looks like mautrix go & python are the only implementations...which are yours so I'm going to assume you know what you're doing. 😄

**Pull request merged**